llm_mining_core/utils/requests_utils.py

```
@@ -48,27 +52,45 @@ def send_miner_request(config, miner_id, model_id):
-        warning_message = response.text.split(warning_indicator)[1].strip('"')
-        print(f"WARNING: {warning_message}")
-        return None, None
+    retry_strategy = Retry(
+        total=0,  # Disable retries
+        connect=0,  # Disable connect retries
+        read=0,  # Disable read retries
+        redirect=0,  # Disable redirect retries
+        status=0,  # Disable status retries
+        status_forcelist=[],  # No status codes to force retry
+        allowed_methods=[]  # Disable retries on all methods
+    )
+    adapter = HTTPAdapter(max_retries=retry_strategy)
+    
+    with requests.Session() as session:
+        session.mount("http://", adapter)
+        session.mount("https://", adapter)
 
-    try:
-        data = response.json()
-        end_time = time.time()
-        request_latency = end_time - current_time
-        if isinstance(data, dict):
-            return data, request_latency
-        else:
+        try:
+            response = session.post(url, json=request_data)
+            response.raise_for_status()  # Raise an exception for 4xx or 5xx status codes
+    
+            # Assuming response.text contains the full text response from the server
+            warning_indicator = "Warning:"
+            if warning_indicator in response.text:
+                # Extract the warning message and use strip() to remove any trailing quotation marks
+                warning_message = response.text.split(warning_indicator)[1].strip('"')
+                print(f"WARNING: {warning_message}")
+                return None, None
+    
+            data = response.json()
+            end_time = time.time()
+            request_latency = end_time - current_time
+    
+            if isinstance(data, dict):
+                return data, request_latency
+            else:
+                return None, None
+    
+        except requests.exceptions.RequestException as e:
+            print(f"Error sending request: {e}")
```

let's fail silently, otherwise people will be worried